### ISSUE TITLE
Cannot compile: "Base-level rules cannot contain the parent-selector-referencing  character '&'."

### ISSUE THREAD

**Issue by heavior** [chinchang/hint.css#145]
I'm trying to include hint.css and change its style, but cannot compile it using the compass:

Error: Base-level rules cannot contain the parent-selector-referencing character '&'.\A on line 18 of /bower_components/hint.css/src/hint-mixins.scss, in `arrow-border-color'\A from line 81 of /bower_components/hint.css/src/hint-position.scss\A from line 23 of /bower_components/hint.css/src/hint.scss\A from line 9

**Comment by chinchang** [comment]
Whats your node-sass version?

**Comment by heavior** [comment]
node-sass: 3.9.5
compass:  1.0.3

**Comment by d8vjork** [comment]
```
Message:
    src\sass\hint\hint-always.scss
Error: Undefined variable: "$hintPrefix".
        on line 11 of src/sass/hint/hint-always.scss
>>                      @include set-margin('translateY', -1, -100%);
```

Same here, with gulp-sass, cannot compile

**Comment by ErinPo** [comment]
`Error: Base-level rules cannot contain the parent-selector-referencing character '&'.
        on line 18 of hint-mixins.scss, in `arrow-border-color'
        from line 81 of hint-position.scss
        from line 23 of hint.scss
  Use --trace for backtrace.`

Same here! Anyone figured out how to correct?

**Comment by chinchang** [comment]
I tried out with node-sass 4.5.0. Here is my test case - I made a file `hello.scss` and imported `hint.scss` in it:

```
@import './src/hint.scss';
```

This `hello.scss` compiled successfully.
@heavior @d8vjork @ErinPo is your use case same as mine or I am testing the wrong thing?

**Comment by ErinPo** [comment]
I did
> sudo gem install sass
and then
> sass hint.scss hint.css
on MacOS

**Comment by jptaylor** [comment]
Seems to work with node-sass, but not the gem.
```
sass -v
Sass 3.4.23 (Selective Steve)
```

**Comment by chinchang** [comment]
fixed in 2.5.1
